### PR TITLE
D2K - Fix tiles 58, 365, 366 and 368

### PR DESCRIPTION
**Pull request by MustaphaTR**
Those are some tiles that are found on Ordos1a.

### CODE REVIEW

File: mods/d2k/tilesets/arrakis.yaml [hunk 1]
```diff
@@ -3067,30 +3067,30 @@ Templates:
...
+			0: Sand
+			1: Sand
 	Template@366:
 		Id: 366
 		Images: BLOXICE.R8
 		Frames: 309, 310, 311, 312, 355, 356, 357, 358, 1, 2, 3, 4, 5, 6, 7, 8
 		Size: 4,4
 		Category: Ice-Detail
 		Tiles:
-			0: Sand
```
Comment (ltem) [inline mods/d2k/tilesets/arrakis.yaml]: I would keep this as sand, it doesn't look wrong if units can be placed there

**Review by ltem — COMMENTED** [review]
A PR where it depends (in my opinion) quite a lot on personal preferences :

- 58, 368: I interpret this as a dead soldier on rough ground, so it is kind of a "mark" for me and I would keep it as a `cliff`
- 365: Totally fine with it
- 366: I would keep some pieces as `sand` like this:
```
 S C C S
 C C C C
 S C C C
 S C C S       S...Sand, C...Cliff
```

| |Reachable tiles filled with infantry (except the "neutral" one)| 
|:--------------:|:------------:|
neutral | tile2_0_a
without PR | tile2_1_a
without PR + tanks | tile2_1_b
with PR | tile2_2_a

**Comment by MustaphaTR** [comment]
I'm changing the values to what original game has, now what i want. So i don't think i can shange them because you want/think it is correct.

**Comment by ltem** [comment]
hmm yeah if it is like this I'm fine with it :+1:

**Review by ltem — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ba463321f7cbd2433983a8eb85e0b6357f429d06...0ca404273879ad0174e1eeb97094862bda7a0082)

**Pull request merged**